### ISSUE TITLE
Add a warning about unsaved content when leaving editor.

### ISSUE THREAD

**Issue by Gipzo** [ludumdare/ludumdare#792]
It would be nice to see some warning about unsaved content when user clicked link that will open in same tab.

Moving back in history didn't preserve form content.

**Comment by povrazor** [comment]
Agreed. it's very unsafe right now, and there is currently no client-side backup in case of user error (accidental tab close).
